Proofreading translations

overwritten opened the pull request:
Changed a few files + added the PSD for the loading screens.

overwritten committed:
Proofreading translations

Changed a few files + added the PSD for the loading screens.

overwritten committed:
Fixed three awkward lines in UI.txt

Title says all

lekluge commented:
I need to check them Ingame First. Pretty Sure some of them too many characters and would display pretty ugly

overwritten committed:
File Cleanup for HF Patch

Fixed a lot of issues + Made a Birthdays.txt and RandomNames.txt + Cleaned up some of the files and translations, should be mostly consistent

There may be a weird bug with sex positions just fucking vanishing idk about that so if it happens just click in the general spot, they're still there, you're just playing blind

overwritten commented:
I also fixed a couple lines that were too long, only one that's one character off is 'Blowjob Master' but everything else works great!
